### DISCUSSION TITLE
I didnt get my benefits

### DISCUSSION THREAD

**Discussion by Hxnnafai** [community/community#139951] [GitHub Education]
### Hi everyone,

![{13C437FC-45ED-42B1-A206-2207D77555B2}](https://github.com/user-attachments/assets/21f7cc5b-9c55-4c43-b71a-1f07249aa1b9)
i aproved 3 days ago but i still didnt get my benefits

**Comment by davevad93** [comment]
Hi @Hxnnafai ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Hxnnafai was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by AbdulAhad365** [comment]
i think this is just a scam, they are using your personal information
